lms/djangoapps/instructor_task/models.py

```diff
@@ -283,6 +288,14 @@ def links_for(self, course_id):
             # Django's FileSystemStorage fails with an OSError if the course
             # dir does not exist; other storage types return an empty list.
             return []
+        except BotoServerError as ex:
+            logger.exception(
```

Comment:
oh, wait, do you really want `logger.exception` here, or `logger.error`?

Reply:
Yes, you are right, `logger.error` makes more sense.